examples/sample-layers/label-layer/label-utils.js

```diff
@@ -21,48 +15,37 @@ function setTextStyle(ctx, fontSize) {
 export function makeTextureAtlasFromLabels(gl, {data, getLabel = x => x, fontSize = 48}) {
   const canvas = document.createElement('canvas');
```

Comment:
So we are creating a new canvas + 2d context every time text is changed. Is there a need to cache the canvas?